developer-tooling/scripts/optimize_runpython.py

```
@@ -0,0 +1,295 @@
+        if '0021_auto_20201020_0908_squashed_0026_auto_20201023_1228' in f:
+            continue
+        if (
+            '0108_alter_purchaseorder_link_and_more_squashed_0109_alter_purchaseorderextraline_link_and_more'
+            in f
+        ):
+            continue
+        target_files.append(f)
+
+    if '--reset' in sys.argv and target_files:
+        print('Reset flag detected. Reverting local changes on squashed files first...')
+        subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)